linux/network/proxy.sls

```diff
@@ -1,25 +1,85 @@
 {%- from "linux/map.jinja" import network with context %}
 {%- if network.enabled %}
-
 {%- if grains.os_family == 'Debian' %}
 
-{%- if network.proxy.host == 'none' %}
-
-/etc/profile.d/proxy.sh:
-  file.absent
-
-/etc/apt/apt.conf.d/95proxies:
-  file.absent
-
+{%- if network.proxy.get('enabled', 'True') or network.proxy.host != 'none' %}
+{%- if network.proxy.host %}
+{%- set fallback_default = network.proxy.host if not network.proxy.get('port', None) else network.proxy.host~":"~network.proxy.port %}
 {%- else %}
-
+{%- set fallback_default=None %}
+{%- endif %}
+#APT
+{%- if network.proxy.apt.enabled %}
```

Comment:
1, apt is defined in map.jina to avoid this
2, is enabled by default in map jinja to keep behaviour backward compatible

Personaly I would rather have it False by default.